acceptance_tests/test_wfs.py

```diff
@@ -1,7 +1,17 @@
-from xml.etree import ElementTree
+from c2cwsgiutils.acceptance.connection import CacheExpected
 
 
 def test_get_feature(connection):
-    answer = connection.get_xml('?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAME=poi&featureId=poi.1')
-    features = answer.findall(".//{http://localhost:8380/wfs?service=WFS&amp;version=2.0.0&amp;request=DescribeFeatureType&amp;typeName=tiger%3Apoi}tiger:poi")
+    """test the WFS connection -- cnx"""
+    answer = connection.get_xml(
+        url="ows?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=poi&featureId=poi.1",
+        cache_expected=CacheExpected.DONT_CARE,
+        cors=False,
+    )
+    print(connection.base_url)
```

Comment:
I don't think the print is necessary here, is it for debug purpose?

Reply:
removed.